components/SocialShareButtons/index.tsx

```diff
@@ -0,0 +1,40 @@
+import Image from 'next/image';
+import {
+  FacebookShareButton,
+  LinkedinShareButton,
+  TwitterShareButton
+} from 'react-share';
+
+const SocialIcon = ({ src, alt }: { src: string; alt: string }) => (
```

Comment:
You should not use SVGs as an Image. Svg  is a vectoral component. You should use as a Svg. Please use react-feather icons. It is already installed. Or you can setup svgr and use custom svg icons.